docs/content/development/extensions-core/namespaced-lookup.md

```diff
@@ -250,3 +270,7 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
   "pollPeriod":600000
 }
 ```
+
+# Introspection
+
+Cached namespace lookups have introspection points at `/keys` and `/values` which return a complete set of the keys and values (respectively) in the lookup.
```

Comment:
in addition to to get `/keys` and `/values` you are exposing `/` that returns all the map.